developer-docs-site/docs/concepts/governance.md

```diff
@@ -45,3 +45,24 @@ sources={{
 :::tip
 Each stake pool can only be used to vote on each proposal exactly once.
 :::
+
+## Who can resolve
+- Anyone can resolve an on-chain proposal that has passed voting requirements. To resolve a proposal, you can use the `aptos governance execute-proposal` command from Aptos CLI. 
+
+## Aptos Improvement Proposals (AIPs)
+
+AIPs are proposals created by the Aptos community or the Aptos Labs team to improve the operations and development of the Aptos chain. 
+To submit an AIP, create an issue in [`Aptos Foundation's github repository`](https://github.com/aptos-foundation/AIPs/issues). 
+To keep up with new AIPs, check the #aip-announcements channel on Aptos' discord channel. 
+To view and vote on on-chain proposals, go to [`Aptos' Governance website`](https://governance.aptosfoundation.org/). 
+
+## Technical Implementation of Aptos Governance
+The majority of governance logic are in [`aptos_governance.move and voting.move`](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources). 
+The `aptos_governance` module outlines how users can interact with Aptos Governance. It's the external-facing module of the Aptos on-chain governance process and contains logic and checks that are specific to Aptos Governance.
+The `voting` module is the Aptos governance standard that can be used by DAOs on the Aptos chain to create their own on-chain governance process.
+
+If you are thinking about creating a DAO on Aptos, you can refer to `aptos_governance`'s usage of the `voting` module as an example. 
```

Comment:
Can you add more details on the functionalities of the voting module and may be just simple explanation of how a DAO can leverage them? We'll add a more detailed tutorial soon.